types/use-analytics/index.d.ts

```diff
@@ -1,32 +1,33 @@
 // Type definitions for use-analytics 0.0
-// Project: https://github.com/DavidWells/analytics/tree/master/packages/use-analytics
-// Definitions by: Harry Cruse <https://github.com/CruseCtrl>
+// Project: https://github.com/DavidWells/analytics
+// Definitions by: Oliver Alonso <https://github.com/boxgames1>
```

Comment:
We should merge these, so we can both be authors :)

```suggestion
// Definitions by: Harry Cruse <https://github.com/CruseCtrl>
//                 Oliver Alonso <https://github.com/boxgames1>
```